sdk/documentintelligence/Azure.AI.DocumentIntelligence/src/DocumentModelAdministrationClient.cs

```diff
@@ -0,0 +1,507 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading.Tasks;
+using System.Threading;
+using System;
+using Azure.Core;
+
+namespace Azure.AI.DocumentIntelligence
+{
+    public partial class DocumentModelAdministrationClient
+    {
+        /// <summary> Gets operation info. </summary>
+        /// <param name="operationId"> Operation ID. </param>
+        /// <param name="cancellationToken"> The cancellation token to use. </param>
+        /// <include file="Generated/Docs/DocumentModelAdministrationClient.xml" path="doc/members/member[@name='GetOperationAsync(Guid,CancellationToken)']/*" />
+        public virtual async Task<Response<OperationDetails>> GetOperationAsync(string operationId, CancellationToken cancellationToken = default)
```

Comment:
The service incorrectly defines the parameter `operationId` to be a `Guid` in the spec, but that's not correct. Because of this I had to manually add these `GetOperation` methods accepting a `string` and hide the generated ones.